### DISCUSSION TITLE
How do triggers work?

### DISCUSSION THREAD

**Discussion by AHMFLibrary** [MCLD/greatreadingadventure#1014] [Help!]
Please help - I can't figure out how to add badges and triggers correctly (it is different than in the older version I used).  I have added 3 triggers with badges so far - 120 points, 240 points, and 360 points. However, when I created a test account and entered 120 minutes read, the system awarded all 3 badges at once. What am I doing wrong? 
I attempted to put some screenshots below, showing how I am setting up the triggers. 
Gina

![image](https://github.com/MCLD/greatreadingadventure/assets/133135413/202536f6-b866-4709-ae06-78869eec57de)

![image](https://github.com/MCLD/greatreadingadventure/assets/133135413/0d12257c-0c4c-46e3-ad87-99092e8c54db)

**Comment by k7hpn** [comment]
Hello @AHMFLibrary - I apologize for getting back to you so late. It appears GitHub was not sending me notifications of discussion posts and so I am only just seeing this now.

Your configuration of triggering on **Must have this many points** set to 120 looks correct, however in the second screenshot under **Award points** you are issuing the participant 120 points for getting this trigger. So what's happening is:

1. Participant enters 120 points.
2. This trigger activates based on your **Must have this many points** setting
3. When the trigger is awarded, it's giving the participant another 120 points based on the **Award points** setting
4. Now the participant has 240 points which is probably setting off your next trigger.

If you clear out the **Award points** field on all your triggers you should be golden.

I apologize again for not responding sooner. :(
